### PR TITLE
fix itms upload failed in mac

### PR DESCRIPTION
**Pull request by pedia**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

#16596

### Description
Copy running command from XCode uploader, since It's only worked in Mac.

### Testing Steps

```shell
fastlane beta
```

### CODE REVIEW

**Comment by pedia** [comment]
@janpio 
Wish help, this PR comming :-;

File: fastlane_core/lib/fastlane_core/itunes_transporter.rb [hunk 1]
```diff
@@ -415,6 +449,9 @@ def upload(app_id, dir)
       UI.verbose(@transporter_executor.build_upload_command(@user, 'YourPassword', actual_dir, @provider_short_name))
 
       begin
+        if Helper.mac?
+          Dir.chdir('/usr/local/itms')
```
Comment (janpio) [inline fastlane_core/lib/fastlane_core/itunes_transporter.rb:453]: I think this might have messed with the tests.

Comment (pedia) [reply fastlane_core/lib/fastlane_core/itunes_transporter.rb]: My bad. Test OK after remove chdir.

**Review by freedombonbon — APPROVED** [review]

**Comment by Niek** [comment]
Fixed in https://github.com/fastlane/fastlane/pull/16689

**Comment by joshdholtz** [comment]
@pedia Thanks so much for working on a fix for this! We ended up going with another solution but I really appreciate your time that you put into this ❤️

**Pull request closed without merging**